HelpSource/Classes/UnitTest.schelp

```diff
@@ -263,6 +263,20 @@ code::
 )
 ::
 
+METHOD:: debug
+Supply some debugging information relevant to the currently running
+test case.  This will be displayed immediately preceding any details
+which may be displayed through use of the code::details:: argument of
+link::#-passed:: and link::#-failed::.
+
+ARGUMENT:: text
+The debugging text.
+
+DISCUSSION::
+code::
+this.debug("Some useful information");
```

Comment:
I think you could adapt an example from the Python guide to better illustrate usage

Reply:
Done.